pepper-apis/src/main/java/org/broadinstitute/ddp/util/GsonCreateUtil.java

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.ddp.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Util methods helping to create simple Json docs
+ */
+public class GsonCreateUtil {
+
+    public static String createJsonIgnoreNulls(String name1, Object value1) {
+        Map<String, Object> map = new HashMap<>();
+        map.put(name1, value1);
+        return  GsonUtil.standardGson().toJson(excludeNullValues(map));
+    }
+
+    public static String createJsonIgnoreNulls(String name1, Object value1, String name2, Object value2) {
```

Comment:
The `Map.of()` static method lets you create maps with arbitrary number of pairs, as long as you pass in even number of arguments. E.g.
```
Map.of(key1, val1)
Map.of(key1, val1, key2, val2)
```
I recommend just having one version of `createJsonIgnoreNulls()` method and have that take in a Map as parameter. Callers can use `Map.of()` to create the map they need.

Reply:
The problem with Map.of() that it not allows pairs with null values - it throws NPE. I also thought to use Map.of initially.
But I want to exclude fields with null values in result payload: for example, to exclude errorMessage=null in case if resultType='SUCCESS' (so that the returned payload will be {'resultType': 'SUCCESS'}.

Reply:
Ah I see. Didn't know about issue with nulls!